### PR TITLE
BA-000.2 - Add and Update CONTRIBUTING.md to yna_backend [fixes #2]

### LINKED ISSUE #2: BA-000.2 - Add and Update CONTRIBUTING.md to yna_backend
Copy the [CONTRIBUTING.md](https://github.com/Young-N-Ambitious/yna_frontend/blob/main/CONTRIBUTING.md) from the yna_frontend and add it to the yna_backend. Make sure to update the docs so that where it says "yna-frontend", it should now say "yna_backend"

### PR DESCRIPTION
**Pull request by najeebkp**
### BA-000.6 - Add and Update CONTRIBUTING.md to yna_backend [fixes #2]

### CODE REVIEW

**Review by stef-codes — DISMISSED** [review]
Thank you for making these changes!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+1. First you make a fork of the repo so that you can use your fork to update the
+   project.
+   1. After forking the repo in your local terminal:
+      1. create a YNA directory by running `mkdir YNA`
+      2. cd into the YNA directory by running `cd YNA`
+      3. Clone the project to your local terminal by running git clone
+         `git@github.com:{YOUR_GITHUB_USERNAME}/yna_backend.git`
+      4. cd into the YNA backend directory by running `cd yna_backend`
+      5. Now run `npm install` to install all of the project dependencies.
```
Comment (Sydnee510) [inline CONTRIBUTING.md:41]: remove` npm install` and add `bundle install`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   2. Now that you are inside the backend directory, you need to create a
+      branch.
+      1. To create a branch run: git checkout -b
+         FD-[#<issue_number>]<commit_type>-<description/name>
```
Comment (Sydnee510) [inline CONTRIBUTING.md:45]: the branch syntax for backend should be BA-[#<issue_number>-<description/name>
**Example branch name: BA-001-enable-cors**

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+### How To Contribute - Pull requests
+
+1. After adding your code you will need to add and commit your changes.
+   1. Run `git status` - to see the files you changed and confirm thats what you
+      want.
+   2. Run `git add .` to add all of the files or `git add <file name>` to add a
+      specific file.
+   3. Once you know and added what files you want to commit. Commit your changes
+      by running `git commit -m "FD-[#<issue_number>]<commit_type>:description"`
```
Comment (Sydnee510) [inline CONTRIBUTING.md:72]: `git commit -m "BA-[#<issue_number>]<commit_type>:description"`

**Review by Sydnee510 — CHANGES_REQUESTED** [review]
hey can you address these small changes @najeebkp

**Comment by najeebkp** [comment]
changes made !

**Review by Sydnee510 — APPROVED** [review]
+1

**Comment by Sydnee510** [comment]
Task linked: [CU-2y49agp BA-000.2 - Create Contributor.md](https://app.clickup.com/t/2y49agp)

**Review by stef-codes — APPROVED** [review]
Thanks for adding this and making the changes!

**Pull request merged**